tests/portfolio/test_ledger_entries.py

```diff
@@ -0,0 +1,43 @@
+import os
+import pytest
+from playwright.sync_api import Page, expect
+from vega_sim.service import VegaService
+from datetime import datetime, timedelta
+from conftest import init_vega
+from fixtures.market import setup_continuous_market
+
+@pytest.fixture(scope="module")
+def vega(request):
+    with init_vega(request) as vega:
+        yield vega
+
+
+@pytest.fixture(scope="module")
+def continuous_market(vega):
+    return setup_continuous_market(vega)
```

Comment:
I don't think we need a market created for this test?

We could just use vega, like in the test_navigation.

Reply:
done